backend/inspirehep/editor/views.py

```diff
@@ -379,3 +368,58 @@ def remove_editor_lock(endpoint, pid_value):
     )
     editor_soft_lock.remove_lock()
     return jsonify(success=True)
+
+
+@blueprint.route("/authorlist/url", methods=["POST"])
+@login_required
+def authorlist_url():
+    """GROBID extraction from PDF"""
+
+    url = request.json["url"]
+    api_path = "api/processHeaderDocument"
+    kwargs_to_grobid = {"includeRawAffiliations": "1", "consolidateHeader": "1"}
+
+    try:
+
+        parsed_authors = post_pdf_to_grobid(url, api_path, **kwargs_to_grobid)
+        authors_normalized_affs = normalize_affiliations_for_authors(parsed_authors)
+        return jsonify(authors_normalized_affs)
+
+    except Exception as err:
+        return jsonify(status=400, message=" / ".join(err.args)), 400
+
+
+def post_pdf_to_grobid(url, grobid_api_path, **kwargs):
+
+    document = requests.get(url, headers={"Content-Type": "application/pdf"}).content
+    data = {"input": document}
+    data.update(kwargs)
+    grobid_url = current_app.config["GROBID_URL"]
+    response = requests.post(urljoin(grobid_url, grobid_api_path), files=data)
+    response.raise_for_status()
+
+    parsed_authors = [
+        author["author"] for author in GrobidAuthors(response.text).parse_all()
+    ]
+    if not response:
+        return
+    return {"authors": parsed_authors}
```

Comment:
We do the following:
* move this function to https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/matcher/api.py 
* name it `get_affiliations_from_pdf`
* no need to pass the `grobid_api_path` param